packages/vue-language-core/src/languageModule.ts

```diff
@@ -2,20 +2,49 @@ import type { Language } from '@volar/language-core';
 import { posix as path } from 'path';
 import { getDefaultVueLanguagePlugins } from './plugins';
 import { VueFile } from './sourceFile';
-import { VueCompilerOptions } from './types';
 import * as sharedTypes from './utils/directorySharedTypes';
 import type * as ts from 'typescript/lib/tsserverlibrary';
 import { resolveVueCompilerOptions } from './utils/ts';
 
+// tried to do something similar to
+// vue compiler-sfc
+// https://github.com/vuejs/core/blob/bdf3492aee2a970ab7e73cf833d729679c731111/packages/compiler-sfc/src/script/resolveType.ts#L845
+// below utils copied from compiler-sfc util script, can it be shared?
+// https://github.com/vuejs/core/blob/bdf3492aee2a970ab7e73cf833d729679c731111/packages/compiler-sfc/src/script/utils.ts#LL83C1-L100C2
+const identity = (str: string) => str;
+const fileNameLowerCaseRegExp = /[^\u0130\u0131\u00DFa-z0-9\\/:\-_\. ]+/g;
+const toLowerCase = (str: string) => str.toLowerCase();
+
+function toFileNameLowerCase(x: string) {
+	return fileNameLowerCaseRegExp.test(x)
+		? x.replace(fileNameLowerCaseRegExp, toLowerCase)
+		: x;
+}
+
+export function createGetCanonicalFileName(useCaseSensitiveFileNames: boolean) {
+	return useCaseSensitiveFileNames ? identity : toFileNameLowerCase;
+}
+let moduleCache: ts.ModuleResolutionCache | null = null;
+function getModuleResolutionCache(
+	ts: typeof import("typescript/lib/tsserverlibrary"),
+	options: ts.CompilerOptions
+) {
+	if (moduleCache === null) {
+		moduleCache = ts.createModuleResolutionCache(
+			process.cwd(),
```

Comment:
The path isn't correct. 
It need to from `host.getCurrentDirectory()`

Reply:
Yea I wasn't sure what path it was looking for exactly. compiler-sfc uses process.cwd so went with that.